account_move_partner_bank_source/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = ["res.partner", "bank.account.mixin"]
+
+    bank_account_id = fields.Many2one(
+        company_dependent=True,
```

Comment:
I guess we can set `company_depenedent=True` in the mixin instead, and remove this field override. Not sure if I'm missing something.